cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_controller_test.go

```diff
@@ -34,233 +34,252 @@ import (
 	fakekube "k8s.io/client-go/kubernetes/fake"
 )
 
-type testControllerConfig struct {
```

Comment:
Just comment: I see the changes as an improvement/simplification of the code. Yet, some functions are still too long given the nature of defining k8s types in a formatted way. Wondering if it would help to create more but smaller functions/methods and call them instead. Since, each refactoring PR is moving a lot of code (even when it's just indenting blocks +/-1. Would be nice to minimize changes.

Reply:
> Yet, some functions are still too long given the nature of defining k8s types in a formatted way.

Are you referring to `mustCreateTestConfigs` - I deliberately put everything into that function to stop the previous sprawl. But now that I have reconciled all the usages I'll least split this up a little and resurrect the `makeLinkedNodeAndMachine`.

Reply:
Done in 49d36503b...b4cff8a56

Reply:
This is now split up again - I resurrected `makeLinkedNodeAndMachine`.